tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -60,6 +60,10 @@ def _run(self):
                     SyncObject(data=json.dumps(raw_data))
                 sync_object.save()
 
+                gluster_brick_dir = NS.gluster.objects.\
+                                     GlusterBrickDir()
+                gluster_brick_dir.save()
```

Comment:
as mentioned above :
My idea was to use the same object to store additional attributes related to gluster brick dir. you were hinting at having a list of user defined brick paths in future, which would also come under this object as a new attribute like: "user_brick_paths"
So in that case user might change the brick paths in between so it has to be synced. hence its placed here.
Please provide your thoughts